catroid/src/main/java/org/catrobat/catroid/cloudmessaging/CloudMessaging.java

```diff
@@ -28,23 +28,38 @@
 import android.app.PendingIntent;
 import android.content.Context;
 import android.content.Intent;
+import android.content.SharedPreferences;
 import android.media.RingtoneManager;
 import android.net.Uri;
+import android.preference.PreferenceManager;
+import android.support.annotation.VisibleForTesting;
+import android.telephony.TelephonyManager;
+import android.util.Log;
 
+import com.google.firebase.messaging.FirebaseMessaging;
+
+import org.catrobat.catroid.BuildConfig;
 import org.catrobat.catroid.R;
+import org.catrobat.catroid.ui.SettingsActivity;
 import org.catrobat.catroid.utils.StatusBarNotificationManager;
 
 public class CloudMessaging {
 
+	private static final String TAG = CloudMessaging.class.getSimpleName();
+
+	public static final String DEFAULT_TOPIC = "DEFAULT_TOPIC";
+
 	private final Context context;
 	private final NotificationBuilderProvider notificationBuilderProvider;
 	private final NotificationManager notificationManager;
+	private static boolean isCloudNotificationsEnabled;
```

Comment:
Currently, `isCloudNotificationsEnabled` is not needed because the default value of `SETTINGS_CLOUD_NOTIFICATIONS` is false anyway, I guess.